### PR TITLE
Add sideci.yml rule

### PR DESCRIPTION
**Pull request by akito19**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -7,6 +7,10 @@ def index
     @users = User.all
   end
 
+  def defualt
+    puts 'typo!'
```
Comment (akitodevnull) [inline app/controllers/users_controller.rb]: __[RuboCop]__

Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -7,6 +7,10 @@ def index
     @users = User.all
   end
 
+  def defualt
```
Comment (akitodevnull) [inline app/controllers/users_controller.rb]: __[Misspell]__

"defualt" is a misspelling of "default"

Comment (akito19) [inline app/controllers/users_controller.rb]: __[development][Misspell]__

"defualt" is a misspelling of "default"

Comment (akitodevnull) [inline app/controllers/users_controller.rb]: __[staging][Misspell]__

"defualt" is a misspelling of "default"

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (akitodevnull) [inline app/controllers/users_controller.rb]: __[staging][RuboCop]__

Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.

**Pull request merged**